CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/ExpressionRequestHandler.swift

```diff
@@ -0,0 +1,63 @@
+//
```

Comment:
Are we using this class anywhere in our code? As far as I can recall, we are making direct call to expression api.

Reply:
We are not, I added it as a part of adding all the APIs. I'll leave it there in case it might be needed in the future.